.github/workflows/ampd-handlers-docker.yaml

```diff
@@ -0,0 +1,212 @@
+name: ampd-handlers - Build and push images to ECR
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'ampd-handlers/**'
+      - 'packages/ampd-sdk/**'
+      - '.github/workflows/ampd-handlers-docker.yaml'
+  workflow_dispatch:
+    inputs:
+      handler:
```

Comment:
Are they a finite known list? If yes, we could improve UX by doing like we did here: https://github.com/axelarnetwork/axelarate/actions/workflows/sync-private-repo.yml

This way we can't select a wrong handler or mistype it